Gruntfile.js

```diff
@@ -82,10 +82,9 @@ function configureGrunt(grunt) {
     grunt.registerTask("jasmine-node:test-on-server", function() {
         var done = this.async();
 
-        function afterJasmineNodeCompletes(exitCode) {
-            if (exitCode !== 0) {
-                grunt.fail();
-                return;
+        function preserveExitCode(exitCode) {
```

Comment:
I wrote that name and changed the inner implementation of the function. What I mean now is to abort on error. How about `abortOnError`?